ci/install/main.cc

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iostream>
+
+#include <google/spanner/v1/spanner.pb.h>
+#include <google/bigtable/v2/bigtable.pb.h>
+
+int main() {
+  std::cout << "Hello" << "\n";
```

Comment:
We should at least use a symbol, in case the linkage is broken. Maybe something like:

```C++
  auto creds = grpc::InsecureChannelCredentials();
  auto channel = grpc::CreateChannel("localhost:12345", creds);
  auto stub = google::bigtable::v1::Table::NewStub(channel); // check my math here, can't remember the right stub.
```

Reply:
Done